consensus/src/rand/rand_gen/rand_manager.rs

```diff
@@ -259,7 +274,8 @@ impl<
             )
         );
 
-        let _guard = self.broadcast_aug_data();
+        let certified_data = self.broadcast_aug_data().await;
+        let _guard = self.broadcast_certified_aug_data(certified_data);
```

Comment:
Move this before spawning `verification_task`?

Reply:
why? if we don’t spawn verification task we can’t get certified data

Reply:
Good point. Its just some verification_task needs aug data, but not important